test/react_client/event_helpers_test.dart

```diff
@@ -122,6 +122,12 @@ main() {
       when(nativeKeyboardEvent.metaKey).thenReturn(false);
       when(nativeKeyboardEvent.repeat).thenReturn(false);
       when(nativeKeyboardEvent.shiftKey).thenReturn(false);
+      when(nativeKeyboardEvent.detail).thenReturn(1);
+      when(nativeKeyboardEvent.view).thenReturn(window);
+      when(nativeKeyboardEvent.getModifierState('Alt')).thenAnswer((_) {
+        calls.add('getModifierState');
+        return false;
+      });
```

Comment:
Why do none of the `SyntheticKeyboardEvent` tests in this file check `code` and `which`?